cloudlift/deployment/ecs.py

```diff
@@ -429,6 +440,11 @@ def update_task_definition(self, task_definition):
                 'memory' : task_definition.memory or u'',
 
             }
+        response = self._client.describe_task_definition(
```

Comment:
Can this additional api call be avoided by getting the task definition with tags earlier in the flow?

Reply:
api call removed and tags are sourced from different function